tornettools/plot.py

```diff
@@ -259,7 +259,7 @@ def __plot_transfer_error_rates(args, circuittype, torperf_dbs, tornet_dbs, erro
     dbs_to_plot = torperf_dbs + tornet_dbs
 
     __plot_cdf_figure(args, dbs_to_plot, f"transfer_error_rates_{error_key}.{circuittype}",
-                      xlabel=f"{circuittype} Transfer Error Rate (\%): Type={error_key}")
+                      xlabel=f"{circuittype} Transfer Error Rate (\\%): Type={error_key}")
```

Comment:
Just commenting here since I noticed this before on the benchmark graphs, I think all `\` characters can be removed from this line since `%` doesn't need to be escaped.

For example, see the old behavior:

![transfer_error_rates_ALL exit](https://user-images.githubusercontent.com/3708797/154295005-52399069-81d9-48ef-83c6-ccaf09356ea2.png)

Reply:
I'm fairly certain I've seen the graph generation fail without the `\` here, due to `%` being a control character. I wouldn't be opposed to trying to fix it s.t. the `\` literal doesn't show up in the output, but would rather have that be a separate PR.

This change itself should be a no-op. IIUC, since `\%` isn't a valid escape sequence, it is emitted as a literal `\%` anyway. `\\%` just lets the linter know that's really what we mean.

```
>>> print("\%")
\%
>>> print("\\%")
\%
>>> '\%' == '\\%'
True
```

Reply:
> % doesn't need to be escaped unless you're using old-style format strings

To clarify - I think the unescaped % has caused trouble further down in the toolchain (tex? ps? pdf?)

OTOH I just tried removing the escape and it seems to work... but it might depend what version of the graphing/rendering tools are installed. @robgjansen do you remember any more about this issue?

https://github.com/shadow/tornettools/pull/68

Reply:
I think it might only show up when the string is being processed through latex, because in latex the `%` is the character to start a line comment.

I wasn't able to quickly determine if it works without the escape because latex tools are not installed on the CI machine, or if because we've changed the scripts such that the strings are not passed through the latex engine anymore.

Reply:
Maybe the error was when building the *paper* that incorporates the graphs?